config/locales/en.yml

```
@@ -638,7 +749,7 @@ en:
       admin: "Admin"
     badges:
       admin: "Administrator"
-      mod: "Moderator"
+      mod: "Mod"
```

Should be "Moderator", not "Mod"

```suggestion
      mod: "Moderator"
```